### PR TITLE
fix: use empty histogram if the width is 0

### PR DESCRIPTION
**Pull request by hi-rustin**
Before:

https://github.com/hi-rustin/tokio-console-web/assets/29879298/a0bd3147-3f40-428c-a191-07c47b879e4c

After:

https://github.com/hi-rustin/tokio-console-web/assets/29879298/e35ccdf3-a511-448b-8035-15f8d6b80f95

### CODE REVIEW

File: app/histogram/src/lib.rs [hunk 1]
```diff
@@ -79,13 +80,17 @@ impl Percentile {
 /// Deserializes the histogram from the given bytes.
 #[wasm_bindgen(js_name = deserializeHistogram)]
 pub fn deserialize_histogram(bytes: &[u8], width: isize) -> MiniHistogram {
+    if width == 0 {
```
Comment (suyanhanx) [inline app/histogram/src/lib.rs:83]:
```suggestion
    if width <= 0 {
```

I think it should be `<=`? Since this width is an `isize` var.

**Review by suyanhanx — APPROVED** [review]
Thank you!

File: app/histogram/src/lib.rs (hunk 1 above)
Comment (hi-rustin) [reply app/histogram/src/lib.rs:83]: Good catch!

**Comment by hi-rustin** [comment]
Thanks for your review!  💚 💙 💜 💛 ❤️

**Pull request merged**